agent/consul/autopilotevents/ready_servers_events_test.go

```diff
@@ -184,12 +185,24 @@ func TestAutopilotStateToReadyServersWithTaggedAddresses(t *testing.T) {
 		types.NodeID("db11f0ac-0cbe-4215-80cc-b4e843f4df1e"),
 		structs.NodeEnterpriseMetaInDefaultPartition(),
 		structs.DefaultPeerKeyword,
-	).Once().Return(
+	).Times(2).Return(
 		uint64(0),
 		&structs.Node{TaggedAddresses: map[string]string{"wan": "9.8.7.6"}},
 		nil,
 	)
 
+	store.On("NodeService",
+		memdb.WatchSet(nil),
+		"",
```

Comment:
Shouldn't this be a non-empty node name?

Reply:
The node names all seem to be empty (has NodeIDs instead)

Reply:
Ah, its empty because the mocked Nodes a few lines up don't set a `Name`

Reply:
Maybe it would be best to have those return a name and then expect the NodeService calls to pass those names.

Reply:
Yep, I can make that change